### ISSUE TITLE
Consider replacing nixops with krops

### ISSUE THREAD

**Issue by jonasnick** [fort-nix/nix-bitcoin#115]
I just learned about krops (https://tech.ingolf-wagner.de/nixos/krops/) which is simpler than nixops and sufficient for most applications.

* pro
   * native file encryption
   * build happens on target 
       * easier for macOS users
       *  easier to deploy to arm
       * using no substitutes won't slow down your personal computer
* con
    * no easy virtualbox tutorial anymore
        * but we could provide a build with [nixos-generate](https://github.com/nix-community/nixos-generators)

**Comment by nixbitcoin** [comment]
Would it be possible to document both and allow the user to choose? Would the user be able to use both in parallel?

**Comment by nixbitcoin** [comment]
Where do we stand on this? Can I switch from nixops to krops right now and document?

**Comment by jonasnick** [comment]
To be clear, nowadays replacing would mean that we replace nixops in the docs/install.md instructions  with krops. I don't have much experience with krops still besides having it used in one deployment and that worked without any issues. A major upside would be that we could remove the macOS section (krops builds on the target) and remove the VirtualBox section (it'd be the same as installing on any other NixOS target). Moreover we could remove the nixops VirtualBox example because we have a qemu example which is simpler (only dependency is Nix).

So in general I still think it would be an improvement to use krops because most of the complexity (statefulness) is not needed for nix-bitcoin deployments (and I've had lots of troubles with it).

**Comment by nixbitcoin** [comment]
What if we

1. Keep NixOps as recommended deployment method for https://github.com/fort-nix/nix-bitcoin/blob/master/docs/install.md#tutorial-install-and-configure-nixos-for-nix-bitcoin-on-virtualbox
2. Change recommended deployment method to Krops for  https://github.com/fort-nix/nix-bitcoin/blob/master/docs/install.md#tutorial-install-and-configure-nixos-for-nix-bitcoin-on-your-own-hardware

That would also close https://github.com/fort-nix/nix-bitcoin/issues/56

**Comment by jonasnick** [comment]
Another alternative I just came across is [morph](https://github.com/DBCDK/morph). It's also stateless and declarative.

**Comment by jonasnick** [comment]
Just cam across https://github.com/misuzu/nix-simple-deploy, a rewrite of nix-deploy.
